cpu/stm32/periph/pm.c

```diff
@@ -52,7 +52,8 @@
 #define PM_STOP_CONFIG  (PWR_CR1_LPMS_0)
 #elif defined(CPU_FAM_STM32F7)
 #define PM_STOP_CONFIG  (PWR_CR1_LPDS | PWR_CR1_FPDS | PWR_CR1_LPUDS)
-#elif defined(CPU_FAM_STM32WB)
+#elif defined(CPU_FAM_STM32WB) || defined(CPU_FAM_STM32MP1)
+#define PM_STOP_CONFIG  (0)
```

Comment:
Now the `case STM32_PM_STOP:` will get enabled further down - is this intentional?

Reply:
I took a bite more time to answer this one because I do not feel comfortable with power management.
As I understand STM32_PM_STOP and STM32_PM_STANDBY are always defined:
https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32/include/periph_cpu.h#L138
Thus `#ifdef STM32_PM_STOP` and `#ifdef STM32_PM_STANDBY` are always true, am I right ?
Then `PM_STOP_CONFIG` must always be defined and as in stop mode, only SLEEPDEEP bit has to be set for the MCU, thus by setting `PM_STOP_CONFIG`, nothing will be done except SLEEPDEEP bit with `cortexm_sleep(deep);`.

As the datasheet says:
> For MCU when the SLEEPDEEP bit in the Cortex®-M4 System Control register is set.
Source: https://www.st.com/resource/en/reference_manual/dm00327659-stm32mp157-advanced-armbased-32bit-mpus-stmicroelectronics.pdf#page=476

However there is also the MPU (Cortex-A7) mode to consider but from my point of view RIOT should not control the power management of the MPU at the moment. Maybe we could consider this later.

What do you think ?